test/app.e2e-spec.ts

```diff
@@ -178,7 +179,13 @@ describe('AppController (e2e)', () => {
         data: '0x7b2264617461223a2274657374227d',
         is_fungible: false,
       },
-      OPTIONS,
+      {
+        ...OPTIONS,
+        params: {
+          ...OPTIONS.params,
+          'fly-from': '0xabc',
```

Comment:
You probably want to make the `fly-` in `fly-from` configurable, as FireFly could be pointed at an ethconnect that's using a different query scheme (such as `kld-from`, or another vendor specific prefix)